examples/with-rollup/README.md

```diff
@@ -0,0 +1,72 @@
+# Turborepo starter with Rollup
+
+This is an official starter Turborepo, showing how Turborepo can be integrated with Rollup for bundling a `ui` package.
+
+## What's inside?
+
+This Turborepo includes the following packages/apps:
+
+### Apps and Packages
+
+- `ui`: a React component library used by the `web` application, compiled with Rollup
+- `web`: a [Next.js](https://nextjs.org) app
+- `eslint-config-custom`: `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
+- `tsconfig`: `tsconfig.json`s used throughout the monorepo
+
+Each package/app is 100% [TypeScript](https://www.typescriptlang.org/).
```

Comment:
"workspace" ?

Reply:
I like it but will stay consistent with our other examples for now.